docs/upgrading.md

```diff
@@ -0,0 +1,66 @@
+# Upgrading Contour to 0.15
+
+## Changes in 0.15
+
```

Comment:
Add an introductory paragraph here.

Contour 0.15 has made several changes to the deployment manifests which you should be aware of.